schema/attestation.graphqls

```diff
@@ -12,13 +12,22 @@ extend type Query {
     """
     The attesting party. 
     """
-    signer: Address
+    source: Address
+    """
+    Filter attestation by producer effectiveAt, expiresAt, or attestation ID. 
+    """
+    filter: AttestationFilter
   ): [Attestation]
     @requiresVehicleToken
     @requiresAllOfPrivileges(privileges: [VEHICLE_RAW_DATA])
 }
 
 type Attestation {
```

Comment:
We should add the signature, so people have the option to verify themself instead of trusting DIS.